src/pudl/etl.py

```diff
@@ -192,24 +189,16 @@ def _etl_ferc1(
         data, keyed by table name.
 
     """
-    ferc1_years = etl_settings.years
-    ferc1_tables = etl_settings.tables
-
-    if not ferc1_years or not ferc1_tables:
-        logger.info('Not loading FERC1')
-        return []
-
     # Compile static FERC 1 dataframes
     out_dfs = _read_static_tables_ferc1()
 
     # Extract FERC form 1
     ferc1_raw_dfs = pudl.extract.ferc1.extract(
-        ferc1_tables=ferc1_tables,
-        ferc1_years=ferc1_years,
+        ferc1_settings=etl_settings,
```

Comment:
It seems more readable to me to use the data source specific name (e.g. `ferc1_settings`) rather than the generic name (`etl_settings`) inside the functions when we can, as it provides more context, and the `etl_settings` name can just pertain to the higher level object that contains all the individual data source settings. I know it was like this already. What do you think?